test/e2e/case38_install_operator_test.go

```diff
@@ -23,7 +23,7 @@ import (
 	"open-cluster-management.io/config-policy-controller/test/utils"
 )
 
-var _ = Describe("Testing OperatorPolicy", Ordered, func() {
+var _ = Describe("Testing OperatorPolicy", Ordered, Label("hosted-as-well"), func() {
```

Comment:
I wonder if you can define the whole thing twice, once with the label, and once without... Or maybe Dale had a suggestion for something similar?

But we can probably just merge this as is, the additional label isn't too bad. Let me know if you're tired of trying random things from our comments :laughing: 

```suggestion
var _ = Describe("Testing OperatorPolicy", Ordered, Label("hosted-mode"), installOperatorTests)
var _ = Describe("Testing OperatorPolicy", Ordered, installOperatorTests)

func installOperatorTests() {
```

Reply:
This is good but why are we afraid of adding a new label?

Reply:
Mainly I just wanted the test filters to be as simple as possible, it makes it more obvious which tests they run. But the new label is probably fine